shared/routes/jobs.routes.v2.ts

```diff
@@ -400,103 +400,39 @@ export const zJobsRoutesV2 = {
-      querystring: ZJobOpportunityRncp,
-      response: {
-        "200": z.array(ZJobsPartnersRecruteurLba),
-      },
-      securityScheme: {
-        auth: "api-apprentissage",
-        access: null,
-        resources: {},
-      },
-    },
-    "/jobs/rome/offres_emploi_lba": {
-      method: "get",
-      path: "/jobs/rome/offres_emploi_lba",
-      querystring: ZJobOpportunityRome,
-      response: {
-        "200": z.array(ZJobsPartnersOffresEmploiLba),
-      },
-      securityScheme: {
-        auth: "api-apprentissage",
-        access: null,
-        resources: {},
-      },
-    },
-    "/jobs/rncp/offres_emploi_lba": {
-      method: "get",
-      path: "/jobs/rncp/offres_emploi_lba",
-      querystring: ZJobOpportunityRncp,
-      response: {
-        "200": z.array(ZJobsPartnersOffresEmploiLba),
-      },
-      securityScheme: {
-        auth: "api-apprentissage",
-        access: null,
-        resources: {},
-      },
-    },
-    "/jobs/rome/offres_emploi_partenaires": {
-      method: "get",
-      path: "/jobs/rome/offres_emploi_partenaires",
-      querystring: ZJobOpportunityRome,
-      response: {
-        "200": z.array(ZJobsPartners),
+        "200": z.object({
+          jobs: z.object({
+            lba: z.array(ZJobsPartnersOffresEmploiLba),
+            france_travail: z.array(ZJobsPartnersOffresEmploiFranceTravail),
+            partners: z.array(ZJobsPartners),
+          }),
+          recruiters: z.array(ZJobsPartnersRecruteurLba),
+        }),
       },
       securityScheme: {
         auth: "api-apprentissage",
         access: null,
         resources: {},
       },
     },
-    "/jobs/rncp/offres_emploi_partenaires": {
+    "/jobs/rncp": {
       method: "get",
-      path: "/jobs/rncp/offres_emploi_partenaires",
+      path: "/jobs/rncp",
       querystring: ZJobOpportunityRncp,
       response: {
-        "200": z.array(ZJobsPartners),
-      },
-      securityScheme: {
-        auth: "api-apprentissage",
-        access: null,
-        resources: {},
-      },
-    },
-    "/jobs/rome/offres_emploi_france_travail": {
-      method: "get",
-      path: "/jobs/rome/offres_emploi_france_travail",
-      querystring: ZJobQuerystringFranceTravailRome,
-      response: {
-        "200": z.array(ZJobsPartnersOffresEmploiFranceTravail),
-      },
-      securityScheme: {
-        auth: "api-apprentissage",
-        access: null,
-        resources: {},
-      },
-    },
-    "/jobs/rncp/offres_emploi_france_travail": {
-      method: "get",
-      path: "/jobs/rncp/offres_emploi_france_travail",
-      querystring: ZJobQuerystringFranceTravailRncp,
-      response: {
-        "200": z.array(ZJobsPartnersOffresEmploiFranceTravail),
+        "200": z.object({
+          jobs: z.object({
+            lba: z.array(ZJobsPartnersOffresEmploiLba),
+            france_travail: z.array(ZJobsPartnersOffresEmploiFranceTravail),
+            partners: z.array(ZJobsPartners),
+          }),
```

Comment:
On avait pas dit qu'on mettait les jobs ensemble, avec possiblement un tri plus pertinent par la suite ? 
```suggestion
          jobs: z.array(zOffreEmploi),
```

Reply:
Aussi, pourquoi ne pas avoir un seul zod pour les offres ?

Reply:
Je pensais qu'on scindait quand même à l'intérieur pour pas mélanger, mais ok je merge. 
J'ai différent zod car je me sers du même modèle que j'adapte suivant la source des données pour les formater dans le modèle finale

Reply:
Mais du coup, le schema doit etre le meme en sortie quelque soit l'offre non ?

Reply:
Par exemple, pour FT, je n'ai pas d'id mongoDB

Reply:
J'ai mergé les offres local et gardé un modèle zod pour FT

Reply:
Il y a juste l'id qui est différent?